.npmignore

```diff
@@ -0,0 +1,6 @@
+test/
```

Comment:
Was not aware of the fallback to the gitignore file.
I don't see any reason to include the test code in the npm package as it takes additional space and will not be used if the package is installed as a dependency.
The main reason for an explicit npm ignore are the `.vscode` files which are ignored by my global gitignore bull would be included to the npm package otherwise.